elements/bulbs-video/elements/video-carousel.js

```diff
@@ -57,7 +57,7 @@ class BulbsVideoCarousel extends BulbsHTMLElement {
       '<bulbs-video-carousel> MUST contain a <bulbs-carousel>'
     );
 
-    this.videoPlayer.addEventListener('ended', this.playerEnded = this.playerEnded.bind(this), true);
+    this.videoPlayer.addEventListener('jw-complete', this.playerEnded = this.playerEnded.bind(this), true);
```

Comment:
The video `ended` fires for ads and content.

BulbsVideo now forwards the jwplayer `complete` event as `jw-complete`, this only fires when content completes, not an ad.